src/xlat-i/interfaceBuilder.h

```diff
@@ -347,12 +347,12 @@ namespace Builder {
       : ret(ret_)
       , name(name_)
       , stackSize(stackSize_)
-      , eattrib(0)
+      , attribs(NULL)
```

Comment:
Minor, but should prefer `nullptr` here and below.

Reply:
I chose `NULL` vs. `nullptr` to maintain internal consistency with the rest of the file's contents, since it uses `NULL` in other places. I could change this one use to `nullptr`, or refactor them all in one go in another PR. Any thoughts on which would be the best way to go?

Reply:
IMHO we should use nullptr in new code, particularly where it is important in constructors and function calls. Various files in the codebase are already internally inconsistent w.r.t. NULL/nullptr.